lib/ruby_smb/smb1/bit_field/nt_create_andx_flags.rb

```diff
@@ -0,0 +1,22 @@
+module RubySMB
+  module SMB1
+    module BitField
+      # The Flags bit-field for an SMB1 SMB_COM_NT_CREATE_ANDX Request as defined in
+      # [2.2.4.64.1 Request](https://msdn.microsoft.com/en-us/library/ee442175.aspx)
+      # [2.2.4.9.1 Client Request Extensions](https://msdn.microsoft.com/en-us/library/cc246332.aspx)
+      class NtCreateAndxFlags < BinData::Record
```

Comment:
If this Bitfield is only ever used in this one packet, you could also just embed this in the packet asa nested struct like so:
https://github.com/rapid7/ruby_smb/blob/master/lib/ruby_smb/smb1/packet/nt_trans/create_request.rb#L16-L26

Reply:
Indeed, it is a good idea. Thanks for the heads-up, I will fix this.

Reply:
I removed this class and added an embedded struct. I also refactored the request and response specs to match the code you pointed out (https://github.com/rapid7/ruby_smb/blob/master/spec/lib/ruby_smb/smb1/packet/nt_trans/create_request_spec.rb). Addressed in d38c671976c78007294a6334ae06b8c244f22b34.